pkg/test/settings.go

```diff
@@ -25,7 +25,7 @@ import (
 	azsettings "github.com/Azure/karpenter/pkg/apis/settings"
 )
 
-type SettingOptions struct {
+type SettingsFields struct {
```

Comment:
why change to Fields and why change Setting to Settings?

Reply:
Since our new module is called `Options` and will have a similar testing mechanism (look at `pkg/test/options.go`), this name is confusing. The options one used `OptionsFields`, so the parallel counterpart here would be `SettingsFields`.